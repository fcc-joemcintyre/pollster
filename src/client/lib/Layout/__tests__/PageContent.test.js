/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { PageContent } from '..';

test ('PageContent renders default component', () => {
  const component = renderer.create (
    <PageContent />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('PageContent with custom content width', () => {
  const component = renderer.create (
    <PageContent maxw='1024px' />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('PageContent with custom content width and padding', () => {
  const component = renderer.create (
    <PageContent maxw='1024px' p='60px 0 20px 0' />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
