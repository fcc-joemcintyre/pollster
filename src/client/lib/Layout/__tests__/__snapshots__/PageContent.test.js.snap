// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent renders default component 1`] = `
.c0 {
  max-width: 768px;
  margin: 0 auto;
  padding: 0 0 0 0;
  overflow: auto;
}

<div
  className="c0"
/>
`;

exports[`PageContent with custom content width 1`] = `
.c0 {
  max-width: 1024px;
  margin: 0 auto;
  padding: 0 0 0 0;
  overflow: auto;
}

<div
  className="c0"
/>
`;

exports[`PageContent with custom content width and padding 1`] = `
.c0 {
  max-width: 1024px;
  margin: 0 auto;
  padding: 60px 0 20px 0;
  overflow: auto;
}

<div
  className="c0"
/>
`;
